src/NuGet.Core/NuGet.Packaging/Strings.resx

```diff
@@ -593,10 +593,10 @@ Valid from:</comment>
     <value>The primary signature and repository countersignature are unrelated.</value>
   </data>
   <data name="Error_NoRepoAllowList" xml:space="preserve">
-    <value>The package signature certificate cannot be trusted as no repository allow list found.</value>
+    <value>The repository allow list is required and was empty or not found.</value>
   </data>
   <data name="Error_NoClientAllowList" xml:space="preserve">
-    <value>The package signature certificate cannot be trusted as no client allow list found.</value>
+    <value>The client allow list is required and was empty or not found.</value>
```

Comment:
The idea is that the client list is an allow list provided by the client (currently only the verify command has this ability with the -CertificateFingerprint option, but in the future this will include any list given by config files) and repo list is an allow list provided by a repository (i.e. when a repository says that their packages have to be signed it gives a list of certificates to allow).

Do you (or @mishra14) have a suggestion on better naming?